_inc/client/components/dev-card/index.jsx

```diff
@@ -8,19 +8,110 @@ import classNames from 'classnames';
 /**
  * Internal dependencies
  */
-import { isDevVersion as _isDevVersion } from 'state/initial-state';
-import { switchPlanPreview } from 'state/dev-version';
+import {
+	isDevVersion as _isDevVersion,
+	userCanViewStats,
+	userIsMaster,
+	userCanDisconnectSite,
+	userCanEditPosts
+} from 'state/initial-state';
 import { getSitePlan } from 'state/site';
-import { canDisplayDevCard, disableDevCard } from 'state/dev-version';
+import { isCurrentUserLinked } from 'state/connection';
+import {
+	switchPlanPreview,
+	canDisplayDevCard,
+	disableDevCard,
+	switchUserPermission
+} from 'state/dev-version';
 import Card from 'components/card';
 
 export const DevCard = React.createClass( {
```

Comment:
I didn't notice this when you originally built this. Maybe we should make a vow, a promise or an oath to never write `React.createClass` again and instead create the components either as functions, if they're stateless, or as ES6 classes. This is so ESLint doesn't continue adding warnings to the list.